orchagent/pbh/pbhrule.cpp

```diff
@@ -52,6 +52,11 @@ bool AclRulePbh::validateAddMatch(const sai_attribute_t &attr)
         return false;
     }
 
+    if (!m_pTable->validateAclRuleMatch(attrId, *this))
```

Comment:
@bingwang-ms It is not needed only in case we guaranty that the m_pTable points to PBH type table, however we do not have this constraint. So the above validation is needed to validate that the match is one of this AclRule implementation supported ones and then we can validate whether we support this match in the table.